features/FEATURE_LWIP/lwip-interface/EthernetInterface.cpp

```diff
@@ -24,12 +24,35 @@ EthernetInterface::EthernetInterface()
 {
 }
 
+bool validate_str(const char *str, const int len)
```

Comment:
Does lwip already verify that the addresses are syntactically correct?

snprintf does come with a code-size cost, though you can just always force a null terminator:
``` cpp
strncpy(_ip_address, ip_address ? ip_address : "", sizeof(_ip_address));
_ip_address[sizeof(_ip_address)-1] = '\0';
```